Sources/SwiftParser/Specifiers.swift

```diff
@@ -772,16 +772,21 @@ extension Parser {
     _ effectSpecifiers: inout S?
   ) -> RawUnexpectedNodesSyntax? {
     var synthesizedAsync: RawTokenSyntax? = nil
-    var synthesizedThrows: RawTokenSyntax? = nil
-    var unexpected: [RawTokenSyntax] = []
+    var synthesizedThrowsClause: RawThrowsClauseSyntax? = nil
+    var unexpected: [RawSyntax] = []
     var loopProgress = LoopProgressCondition()
+
+    func unexpect(_ node: some RawSyntaxNodeProtocol) {
+      unexpected.append(RawSyntax(node))
+    }
+
```

Comment:
I personally think that the code is more readable if you just inline this code. Then it’s easier to follow where `unexpected` is being built up, IMO.